src/events/gitHub/messageCreate.ts

```diff
@@ -0,0 +1,32 @@
+import { on } from "node:events";
+import { setTimeout as wait } from "node:timers/promises";
+import { logger } from "@yuudachi/framework";
+import type { Event } from "@yuudachi/framework/types";
+import { type Message, Events, Client } from "discord.js";
+import { injectable } from "tsyringe";
+import { handleGithubUrls } from "../../functions/gitHub/handler.js";
+import { ASSISTCHANNELS } from "../../util/constants.js";
```

Comment:
`ASSISTCHANNELS` is only support forums, i'm not sure why you restrict it to that? This should generally work in any text channel

Reply:
Delay for assist channels is increased so the files will sent after the autoposter